syntax = "proto3";

option go_package = "github.com/Pwera/Protocol-Buffers-Notes/src/main/generated/go/ContextEnum";
package me.piotr.wera.context;

enum ContextEnum{
    UNDEFINEX = 0;
    GOOD = 1;
    BAD = 2;
}

